recipe/migrations/arch_rebuild.txt

```diff
@@ -509,35 +509,12 @@ rmm
 nccl
 ptxcompiler
 arrow-cpp
-cudadecon
-cudasirecon
-deepmd-kit
-detectron2
-fairscale
 faiss-split
-fcos
-graphvite
-hoomd
 implicit
-kaldi
 kaolin
-libastra
 libcugraph
-mandrake
-meld
-nnpops
 nvidia-apex
 omniscidb
-openmm-plumed
-openmm-torch
-prismatic_split
+openmm
```

Comment:
`openmm` would like to build on other architectures ( https://github.com/conda-forge/openmm-feedstock/issues/8 ) and have discussed this with them before